src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String SOURCE_FILE = "src/main/java/resources/filetest";
+    private final FileReaderImpl fileReader = new FileReaderImpl();
+
+    @Test (expected = NullPointerException.class)
+    public void readFile_Null_notOk() {
+        fileReader.read(null);
+    }
```

Comment:
I think test, where we expect to get NPE it's not a very useful test... If your method does not have a check for null, then it is better not to test it for null, or add a check for null to the service, and then add some test (fix it in all tests)